Rakefile

```
@@ -14,10 +14,11 @@ namespace :db do
       uri = URI.parse(ENV["TEST_DATABASE_URL"])
       host = "-h #{uri.host}"
       user = "-U #{uri.user}"
+      port = "-p #{uri.port}"
     end
 
-    sh "dropdb #{host} #{user} --if-exists transition_test"
-    sh "createdb #{host} #{user} --encoding=UTF8 --template=template0 transition_test"
-    sh "cat db/structure.sql | psql #{host} #{user} -d transition_test"
+    sh "dropdb #{port} #{host} #{user} -w --if-exists transition_test"
```

Looking at the docs for [`dropdb`](https://www.postgresql.org/docs/13/app-dropdb.html) and [`createdb`](https://www.postgresql.org/docs/13/app-createdb.html) in Postgres 13, it looks like we need to specify what these parameters are using flags, e.g.
```suggestion
    sh "dropdb -p #{port} -h #{host} -U #{user} -w --if-exists transition_test"
```

---

The flags are in the variables above on lines 15-17. I copied the pattern that was already there but happy to change them all if it's clearer.

---

Ah, cool. I didn't look further up 🤦‍♂️!